### PR TITLE
Update 23

### PR DESCRIPTION
**Pull request by Rainb0wKey**
Updated guide (1/4)

### CODE REVIEW

**Comment by Rainb0wKey** [comment]
I think this is a more understandable use case for BDFD, if you want, I can add examples

File: src/guides/introduction/beginning.md [hunk 1]
```diff
@@ -1,22 +1,59 @@
 # Beginning
-In this guide, you will learn how to create a Discord bot in BDFD.
+In this guide you will learn how to create your own bot with BDFD.
```
Comment (MineBartekSA) [inline src/guides/introduction/beginning.md]:
``` suggestion
In this guide, you will learn how to create your own bot using BDFD.
```

File: src/guides/introduction/beginning.md [hunk 2]
```diff
@@ -1,22 +1,59 @@
...
+
+## Functions Used
+- [`$nomention`](../../bdscript/nomention.md)
+- [`$reply`](../../bdscript/reply.md)
+
+# Step 1: Creating
+- Go to [Discord Developer Portal](https://discord.com/developers/applications).
 - Click on the *"New Application"* button and provide a name for your application.
 - In top-left corner, click on the hamburger icon (`≡`) and select the *"Bots"* tab.
 - Then, press the *"Add Bot"* button to create a discord bot.
```
Comment (MineBartekSA) [inline src/guides/introduction/beginning.md]: Discord now automatically creates a bot user

File: src/guides/introduction/beginning.md [hunk 3]
```diff
@@ -1,22 +1,59 @@
...
+> Make sure that when you are creating a new bot, your Discord account is signed into the BDFD app. This is so that you don't lose access to your bot in the future.
+- Enter your bot's name and its token *(the one that you copied earlier from the [Discord Developer Portal](https://discord.com/developers/applications))*.
 - If you have a [share code](../other/sharecode.md), toggle on *"Use share code"* and put the code into the text field.
-- After agreeing to the terms of service of both [BDFD](https://botdesignerdiscord.com/tos) & [Discord](https://discord.com/terms), press "Create bot" to create your Discord bot.
+- After agreeing to the Terms of Service of both [BDFD](../terms.md) & [Discord](https://discord.com/terms), press "Create bot" to create your Discord bot.
+
+# Step 2: Inviting
+- Open the BDFD app and select your bot.
+- Click on the *"Invite bot to server"* button.
+- Press the *"Edit invite link permissions"* button and choose the permissions that the bot will have when joined the server.
```
Comment (MineBartekSA) [inline src/guides/introduction/beginning.md]:
``` suggestion
- Press the *"Edit invite link permissions"* button and choose the permissions that the bot will have when joining a server.
```

File: src/guides/introduction/beginning.md [hunk 4]
```diff
@@ -1,22 +1,59 @@
...
+- Then, click on the *"Add your bot to your server"* and selected the server.
```
Comment (MineBartekSA) [inline src/guides/introduction/beginning.md]:
``` suggestion
- Then, click on *"Add your bot to your server"* and selected the server.
```

File: src/guides/introduction/beginning.md [hunk 5]
```diff
@@ -1,22 +1,59 @@
...
+- Click on the *"Continue"* button and your bot will be added to the server.
```
Comment (MineBartekSA) [inline src/guides/introduction/beginning.md]:
``` suggestion
- Click on *the "Continue"* button and your bot will be added to the selected server.
```

File: src/guides/introduction/beginning.md [hunk 6]
```diff
@@ -1,22 +1,59 @@
...
+
+# Step 3: First Code
+To create the first command, you must click on *"Commands"* in the selected bot and press on the *"+ Command creator"*.
```
Comment (MineBartekSA) [inline src/guides/introduction/beginning.md]:
``` suggestion
To create the first command, you must click on the *"Commands"* tab after selecting your bot and press on the *"+ Command creator"*.
```

File: src/guides/introduction/beginning.md [hunk 7]
```diff
@@ -1,22 +1,59 @@
...
+```
+$nomention
+$reply
+Pong!
+```
+- Click on *"Save command"* to save the code.
+
+# Step 4: Bot Online
+### Method 1 - 30 Minutes
+- Click on *"Dashboard"* in the selected bot and press on the *"+ Add free hosting time"* button.
```
Comment (MineBartekSA) [inline src/guides/introduction/beginning.md]:
``` suggestion
- Click on the *"Dashboard"* tab after selecting your bot and press on the *"+ Add free hosting time"* button.
```

File: src/guides/introduction/beginning.md [hunk 8]
```diff
@@ -1,22 +1,59 @@
...
+- Enter the indicated numbers and click *"Confirm"*.
+### Method 2 - 140 Minutes
+- Press on *"Dashboard"* button in the selected bot and click on *"+ Watch ad for 90 minutes of free hosting time"*.
```
Comment (MineBartekSA) [inline src/guides/introduction/beginning.md]:
``` suggestion
- Press on *"Dashboard"* tab after selecting your bot and click on *"+ Watch ad for 140 minutes of free hosting time"*.
- After watching the ad, your bot will receive 140 minutes of hosting time.
```

File: src/guides/introduction/beginning.md [hunk 9]
```diff
@@ -1,22 +1,59 @@
...
+### Method 3 - Premium
+- Use [premium](../../premium/introduction.md#purchasing-premium-points) points to get hosting time without ad.
+> If the bot is not online after these methods, then take a look at this page. [Why my bot is offline?](../../resources/troubleshooting.md#the-bot-is-offline)
```
Comment (MineBartekSA) [inline src/guides/introduction/beginning.md]:
``` suggestion
- Use [premium](../../premium/introduction.md#purchasing-premium-points) points to get hosting time without watching ads.
> If the bot is not online after these methods, then take a look at this page: [Why my bot is offline?](../../resources/troubleshooting.md#the-bot-is-offline)
```

File: src/guides/introduction/beginning.md [hunk 10]
```diff
@@ -1,22 +1,59 @@
...
 
-### Inviting the bot
-To add the bot into a server, [see this guide](../general/interactions/slashCommands/aboutSlashCommands.md#inviting-the-bot).
+## Step 5: Test it!
+Send a `!ping` command to the channel of the server you invited the bot to.
```
Comment (MineBartekSA) [inline src/guides/introduction/beginning.md]:
``` suggestion
Send a `!ping` command in a channel of the server you invited your bot to.
```

**Review by MineBartekSA — CHANGES_REQUESTED** [review]

**Review by MineBartekSA — APPROVED** [review]
Thank you for your hard work!

**Pull request merged**